internal/cmd/match_criteria.go

```diff
@@ -15,7 +15,7 @@ import (
 // checks if a PR matches all filtering criteria
 func PrMatchesCriteria(branch string, prLabels []string) bool {
 	// Check branch criteria if any are specified
-	if !branchMatchesCriteria(branch) {
+	if !branchMatchesCriteria(branch, combineBranchName, branchPrefix, branchSuffix, branchRegex) {
```

Comment:
Where do those value come from during normal execution?

Reply:
They are a set of global variables which is a bit gross here so I brought them into function scope instead -> https://github.com/github/gh-combine/blob/2257a80c561dbcab641d4337993b76da59b763eb/internal/cmd/root.go#L35